### ISSUE TITLE
No gutter mark

### ISSUE THREAD

**Issue by ryuran** [markalfred/SublimeLinter-contrib-stylint#3]
Hello,

I installed SublimeLinter-contrib-stylint in Sublime Text 3 but I haven't gutter marking.
When I run "Lint this view" everything is ok in sublime text console view but no marking in gutters.

No problem with other linters. (jshint, scss-lint)

**Comment by riseremi-4xxi** [comment]
Same problem here. Everything was fine several weeks ago, but now it's broken.
No problems with other linters as well (ESLint)

**Comment by riseremi** [comment]
Same problem here. Everything was fine several weeks ago, but now it's broken.
No problems with other linters as well (ESLint)

Stylint version: 1.5.6 (installed globally)

**Comment by jackbrewer** [comment]
Hi,

A new version of this plugin was released on Friday to address a breaking change in how errors were reported in Stylint 1.5.0. Are you able to confirm you are using v1.1.0 of SublimeLinter-contrib-stylint? Maybe try a re-install of the plugin?

This is what I'm currently seeing with my install – gutter marks appear as expected. It would be interesting to know if you are seeing different.

![screen shot 2016-07-12 at 09 50 10](https://cloud.githubusercontent.com/assets/720908/16760972/0ad23b56-4816-11e6-8a66-57cd94dc6e30.png)

**Comment by ryuran** [comment]
Ok. It's seem fixed with v1.1.0.

**Comment by riseremi** [comment]
Hey, thanks for the response. Reinstalled it.
Everything works fine except `sortOrder` property.

**Comment by fdeneux** [comment]
I seem to have the same problem. I only get warnings about unnecessary semicolons, others are not reported outside of the console.

edit:
Stylint version: 1.5.6
SublimeLinter-contrib-stylint: 1.1.0

**Comment by jackbrewer** [comment]
@fdeneux Unfortunately I can't replicate this issue on my machine. My Sublime console outputs the warnings in a different format, which work fine. Do you have the `reporterOptions` option set in your `.stylintrc`? I've tried playing with it in my setup, but it doesn't seem to affect how my warnings are reporting.

A long shot, but if you're up for delving into the package, could you test if [removing line 27 of linter.py](https://github.com/jackbrewer/SublimeLinter-contrib-stylint/blob/master/linter.py#L27) helps at all? You may need to restart Sublime after making the change.

**Comment by warmrobot** [comment]
Same here.
Win10, Node v5.10.1, stylint 1.5.6
![default](https://cloud.githubusercontent.com/assets/90591/17750898/684a68ac-64cd-11e6-8c74-ab3a0f621a18.PNG)
I can't find linter.py anywhere.

**Comment by fdeneux** [comment]
@jackbrewer I have following reporter options:

``` json
"reporterOptions": {
    "columns": ["lineData", "severity", "description", "rule"],
    "columnSplitter": "  ",
    "showHeaders": false,
    "truncate": true
},
```

Also, removing line 27 from linter.py did not help.

**Comment by markalfred** [comment]
It seems likely that it's caused by not having a column number in the error output.

These all show the gutter mark:

```
1:14 trailingWhitespace error trailing whitespace
2:16 colons error missing colon between property and value
3:33 semicolons error unnecessary semicolon found
```

This doesn't:
```
4 sortOrder error prefer alphabetical when sorting properties
```

**Comment by markalfred** [comment]
Should be resolved via #6, let me know if this is still an issue 👍
